add social links, slack link

samundra opened the pull request:
#### Changes
- add facebook group link
- add slack channel link

samundra committed:
add social links

* facebook link
* slack channel link

geshan commented on about.md:
```diff
@@ -39,4 +39,10 @@ be dealt in a case by case basis.
+
+You can find us on:
+- [Facebook](https://www.facebook.com/groups/109070762572263/)
+- [Slack Channel](http://bit.ly/phpnp-slack)
```
footer :)

geshan commented on about.md:
```diff
@@ -39,4 +39,10 @@ be dealt in a case by case basis.
+## Join us
+
+You can find us on:
+- [Facebook](https://www.facebook.com/groups/109070762572263/)
```
http://bit.ly/phpnp-fb :)

samundra replied:
I think it's good if we keep the facebook link itself than the shortened one.

samundra committed:
add social link

samundra commented:
I removed the previous footer contents and added @geshan suggestion . Visit the pages and scroll to the bottom to the see the footer changes:

Old Footer: http://developers-nepal.github.io/php/
New Footer : http://samundra.github.io/php/

I will create PR when footer layout is finalized.

geshan commented:
merging.
